src/Illuminate/Foundation/Auth/ResetsPasswords.php

```diff
@@ -24,8 +25,16 @@
      */
     public function showResetForm(Request $request, $token = null)
     {
+        $email = '';
+        if (is_null($token) == false) {
```

Comment:
why not `if (! is_null($token))`?

Reply:
I think that this is clearer, or more expressive on what the purpose of the check is at least.